lib/blueprinter/configuration.rb

```diff
@@ -1,16 +1,24 @@
 module Blueprinter
   class Configuration
-    attr_accessor :generator, :method, :sort_fields_by
+    attr_accessor :generator, :if, :method, :sort_fields_by, :unless
+
+    VALID_CALLABLES = %i(if unless)
```

Comment:
Can we `.freeze` this array?
```suggestion
    VALID_CALLABLES = %i(if unless).freeze
```